Increase tolerance of assert check for solve accuracy, or add option to allow failure.

Several Omnsicape users are running into issues with AssertionErrors:

```julia
AssertionError: norm(G * v .- curr) / norm(curr) < 1.0e-6
```

I'm wondering if we can either increase the error tolerance here, make it relative instead of absolute, or add an option to allow failure of this check. My understanding is that the tolerance of 1e-6 was chosen somewhat arbitrarily? When running 100s or 1000s of moving windows, there is more likely to be an edge case where this check fails. If I could tell Circuitscape to allow this check to fail and instead log a warning about potential innacuracies (and maybe their magnitude relative to current density predictions?) around that one moving window, I think that would alleviate a big headache for many Omniscape users. Alternatively, if we can make this tolerance larger, or allow it to be relative, we may end up seeing this issue pop up a lot less, while still enforcing some degree of accuracy in the solutions.

cc @ranjanan @ViralBShah

--- a/Project.toml
+++ b/Project.toml
@@ -1,6 +1,6 @@
 name = "Circuitscape"
 uuid = "2b7a1792-8151-5239-925d-e2b8fdfa3201"
-version = "5.13.1"
+version = "5.13.2"
 
 [deps]
 AlgebraicMultigrid = "2169fc97-5a83-5252-b627-83903c6c433c"

--- a/lib/CircuitscapeMKLPardiso/src/CircuitscapeMKLPardiso.jl
+++ b/lib/CircuitscapeMKLPardiso/src/CircuitscapeMKLPardiso.jl
@@ -50,7 +50,7 @@ function solve_linear_system(factor::MKLPardisoFactorize, matrix, rhs)
     x = zeros(eltype(matrix), size(matrix, 1))
     for i = 1:size(lhs, 2)
         factor(x, mat, rhs[:,i])
-		@assert (norm(mat*x .- rhs[:,i]) / norm(rhs[:,i])) < 1e-6
+		@assert (norm(mat*x .- rhs[:,i]) / norm(rhs[:,i])) < 1e-4
         lhs[:,i] .= x
     end
     lhs

--- a/src/core.jl
+++ b/src/core.jl
@@ -610,7 +610,7 @@ function solve_linear_system(
             G::SparseMatrixCSC{T,V},
             curr::Vector{T}, M)::Vector{T} where {T,V}
     v = cg(G, curr, Pl = M, reltol = T(1e-6), maxiter = 100_000)
-	@assert norm(G*v .- curr) / norm(curr) < 1e-6
+	@assert norm(G*v .- curr) / norm(curr) < 1e-4
     v
 end
 
@@ -618,7 +618,7 @@ end
 function solve_linear_system(factor::SuiteSparse.CHOLMOD.Factor, matrix, rhs)
     lhs = factor \ rhs
     for i = 1:size(rhs, 2)
-		@assert (norm(matrix*lhs[:,i] .- rhs[:,i]) / norm(rhs[:,i])) < 1e-6
+		@assert (norm(matrix*lhs[:,i] .- rhs[:,i]) / norm(rhs[:,i])) < 1e-4
     end
     lhs
 end

